### ISSUE TITLE
Compare unstaged change to a branch and staged change to a branch

### ISSUE THREAD

**Issue by 9527KGG** [eamodio/vscode-gitlens#703]
Currently, the unstaged change, it looks like

![image](https://user-images.githubusercontent.com/12282281/55822720-133f3280-5ab5-11e9-92db-7f575bb9f2b3.png)

For staged change, it looks like
![image](https://user-images.githubusercontent.com/12282281/55822759-27832f80-5ab5-11e9-885f-3d3acf12429d.png)

There is only one group for unstaged and staged. If I right clicked the group, it shows
![image](https://user-images.githubusercontent.com/12282281/55823166-fce5a680-5ab5-11e9-8c4b-558ccc72a700.png)

Feature request part1: please put unstaged change and staged change to different group. 
Feature request part2: please allow to compare unstaged change to a branch. etc. right click unstaged group, I can choose "Select for Compare". Then I can go a branch to choose "Compare with selected"
Feature request part3: please allow to compare staged change to a branch. etc. right click staged group, I can choose "Select for Compare". Then I can go a branch to choose "Compare with selected"

Basically, for part2, it's git command
git diff origin/branch_name all_files.

Basically, for part3, it's git command
git diff --staged origin/branch_name all_files.

If this feature has already be implemented, then sorry for the duplicate request.

**Comment by 9527KGG** [comment]
Currently, to find out all changes I've made on a child/dev branch, comparing to parent/release branch in gitlens. I've to:

1. git add
2. git commit
3. compare local commit to parent branch.

If sth goes wrong, I've to
1. git reset
2. make change to fix the issue.
3. git add
4. git commit
5. compare local commit to parent branch.
or
1. make change to fix the issue.
2. git add
3. git commit
4. compare local commit to parent branch. (The bad thing is: now I've 2 commit #. But I like less commit count).

Thanks.

**Comment by eamodio** [comment]
This is supported in GitLens 11 -- you can compare anything with the working tree.
